samples/client/petstore/cpp-qt5/client/PFXStoreApi.cpp

```diff
@@ -18,86 +17,75 @@
 
 namespace test_namespace {
 
-PFXStoreApi::PFXStoreApi(const QString &scheme, const QString &host, int port, const QString& basePath, const int timeOut) :
-    _scheme(scheme),
-    _host(host),
-    _port(port),
-    _basePath(basePath),
-    _timeOut(timeOut) {
-}
+PFXStoreApi::PFXStoreApi(const QString &scheme, const QString &host, int port, const QString &basePath, const int timeOut)
+    : _scheme(scheme),
+      _host(host),
+      _port(port),
+      _basePath(basePath),
+      _timeOut(timeOut) {}
 
 PFXStoreApi::~PFXStoreApi() {
 }
 
-void PFXStoreApi::setScheme(const QString& scheme){
+void PFXStoreApi::setScheme(const QString &scheme) {
     _scheme = scheme;
 }
 
-void PFXStoreApi::setHost(const QString& host){
+void PFXStoreApi::setHost(const QString &host) {
     _host = host;
 }
 
-void PFXStoreApi::setPort(int port){
+void PFXStoreApi::setPort(int port) {
     _port = port;
 }
 
-void PFXStoreApi::setBasePath(const QString& basePath){
+void PFXStoreApi::setBasePath(const QString &basePath) {
     _basePath = basePath;
 }
 
-void PFXStoreApi::setTimeOut(const int timeOut){
+void PFXStoreApi::setTimeOut(const int timeOut) {
     _timeOut = timeOut;
 }
 
-void PFXStoreApi::setWorkingDirectory(const QString& path){
+void PFXStoreApi::setWorkingDirectory(const QString &path) {
     _workingDirectory = path;
 }
 
-void PFXStoreApi::addHeaders(const QString& key, const QString& value){
+void PFXStoreApi::addHeaders(const QString &key, const QString &value) {
     defaultHeaders.insert(key, value);
 }
 
-void
-PFXStoreApi::deleteOrder(const QString& order_id) {
+void PFXStoreApi::deleteOrder(const QString &order_id) {
     QString fullPath = QString("%0://%1%2%3%4")
```

Comment:
still with %0 starting.

Reply:
I forgot (again) to update the model. I amended my commit: 05c5f8cb5d